index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+    <title>Portfolio - Mike Martin</title>
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+
+    <header>
+        <h1>Mike Martin: Full Stack Developer;</h1>
+        <div class="navLinks">
+            <a href="#portfolioIntroduction">Portfolio</a>
+            <a href="#about">About Me</a>
+            <a href="#contact">Contact</a>
+        </div>
+    </header>
+
+    <section id="home" class="jumbotron">
+        <div class="decorativeBox"><img src="images/profileSuit.jpg" alt="man in profile with sunglasses and beard."></div>
+        <div class="jumbotronTextbox">
+            <h2>some meaningful phrase</h2>
+            <p>A lovely 3-4 line paragraph that will jump us to a certain section. I might make this be a changing banner. every 10secs sounds good.</p>
+            <a href="#portfolioIntroduction" class="jumbotronButton" type="button" value="jumpToSection">What is this wizardry!</a>
+        </div>
+    </section>
+
+    <section id="about" class="aboutMe">
+        <h2>About Me</h2>
+        <p>
+            I am currently training to be a full stack web developer at iO Academy.
+            I come from 15 years experience within the construction industry, with 5 years specialization in controlled demolition.
+            I reached the peak of learning possible to me in the physical world of construction.
+            I can build you anything you need physically and now with the skills I have learnt here at iO, I can build you just about anything virtually!
+            From good looking websites to streamlined database apps, the possibilities are endless.
+            Set the task and watch the magic happen.
+        </p>
+
+    </section>
+    
+    <section class="portfolioHighlightsContainer">
+            <div class="portfolioHighlight">
+                <a href="#"></a>
```

Comment:
are these anchors supposed to wrap the img and h3? currently they do nothing

Reply:
yes they were, i had them nesting the image and the header but charlie suggested i change it to this

Reply:
have nest the content in anchors again